app/decorators/plant_trial_submission_decorator.rb

```diff
@@ -43,7 +43,7 @@ def plant_population
 
   def trait_descriptors
     tds = object.content.step02.trait_descriptor_list.try(:select, &:present?) || []
-    numeric_tds = tds.select { |td| td.to_s.match /\A\d+\z/ }
+    numeric_tds = tds.select { |td| td.to_s.match(/\A\d+\z/) }
```

Comment:
Yes, it seems we have `old_` in two places and `existing_` in two others. So please change to `numeric` to `existing` and also try to replace the `old_` occurrence in `submission.rb` (the other one is in specs, you may ignore that one).

Currently one is not able to call a new TD with a number, since I closed the vocabulary (it's only Trait.name values now). However, I can easily imagine us allowing a free text version, since the TO ontology is rather limiting for our users. Therefore I asked about that. Are existing `tds` stored in the array as integers?